Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:16.04
```

Comment:
Can we stuff this Dockerfile somewhere other than in the root directory? I'm open to new directories to store these configuration type files.